getlyrics/getlyrics.py

```diff
@@ -23,6 +23,14 @@ def prompt(*args):
         sys.stdout = old_stdout
 
 
+def pager(string):
+    import subprocess
+    pager_prog = os.environ['PAGER']
+    handle = subprocess.Popen([pager_prog, '-'], stdin=subprocess.PIPE, shell=False)
+    handle.communicate(string)
```

Comment:
This line returns an error on OSX 10.9 with python 3.4

      File "/Users/steven/workspace/getlyrics/getlyrics/getlyrics.py", line 30, in pager
        handle.communicate(string)
      File "/usr/local/Cellar/python3/3.4.3_2/Frameworks/Python.framework/Versions/3.4/lib/python3.4/subprocess.py", line 941, in communicate
        self.stdin.write(input)
    TypeError: 'str' does not support the buffer interface